driver/cassandra/cassandra.go

```diff
@@ -83,16 +58,16 @@ func (driver *Driver) Close() error {
 }
 
 func (driver *Driver) ensureVersionTableExists() error {
-	err := driver.session.Query("CREATE TABLE IF NOT EXISTS " + tableName + " (version counter, versionRow bigint primary key);").Exec()
-	if err != nil {
-		return err
-	}
 
-	_, err = driver.Version()
-	if err != nil {
-		driver.session.Query(up.String(), versionRow).Exec()
+	if err := driver.session.Query("CREATE TABLE IF NOT EXISTS " + tableName +
+		" (driver_version int," +
+		"version bigint," +
+		"file_name text," +
```

Comment:
I don't think it is relevant to keep the file name. Besides, if you change it in your code you'll have to manually update it here.
Rails doesn't do it! :sheep: